posts/zzz_DO_NOT_EDIT_nest_and_pharmaverseadam/nest_and_pharmaverseadam.qmd

```diff
@@ -0,0 +1,89 @@
+---
+title: "A collaborative triumph: Re-using test data between the NEST and admiral teams"
+author:
+  - name: Joe Zhu
+  - name: Edoardo Mancini 
+description: "This post described one of the first joint efforts between the NEST and admiral teams, where test data from the pharmaverseadam package was used within the NEST framework."
+# Note that the date below will be auto-updated when the post is merged.
+date: "2024-12-01"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [ADaM, TLG, Community]
+# Feel free to change the image
+image: "nest_and_pharmaverseadam.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_nest_and_pharmaverseadam"
+# library(link)
+# link::auto()
```

Comment:
```suggestion
library(link)
link::auto(keep_pkg_prefix = FALSE)
```

This will make the {admiral} into links

Reply:
yes sorry had commented it out as something was not working but will undo that now.